docs/_components/modal.md

```diff
@@ -9,39 +9,41 @@ Modals are a blocking element, they should be used sparingly and only when we ab
 Example:
 
 ```html
-<div class="modal">
+<div class="modal" id="myModal" role="dialog" aria-modal="true" aria-labelledby="dialog-title" aria-describedby="dialog-description">
     <div class="modal__dialog">
         <div class="modal__content">
             <div class="modal__header">
-                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
-                <h4 class="modal__title">A simple example</h4>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close dialog">×</button>
+                <h4 class="modal__title" id="dialog-title">A simple example</h4>
```

Comment:
I know this has always been an `h4` but I'd argue it should really be an `h1`. This would no doubt require a font-size setting for `modal__title`